segmentation_models_pytorch/decoders/sam/model.py

```diff
@@ -0,0 +1,213 @@
+import logging
+from typing import Optional, Union, List, Tuple
+
+import torch
+from segment_anything.modeling import MaskDecoder, TwoWayTransformer, PromptEncoder
```

Comment:
Is it a pip package? probably need to add to reqs

Reply:
just added it to reqs, or should we make it optional?